README.md

```diff
@@ -79,22 +79,48 @@ The email can be filtered, example: `add_filter( 'pb_integrations_multidomain_em
 
 Because this plugin uses the fabulous [onelogin/php-saml](https://github.com/onelogin/php-saml/) toolkit, [many other configuration variables can be tweaked](https://github.com/onelogin/php-saml/#settings).
 
+## Sending logs 
+If you use AWS and wish to log SAML attempts on your server, you will need define some environment variables on the server which is hosting your Pressbooks instance.
+### AWS S3
+Define the following envirnoment variables:
+
+ ```
+  LOG_LOGIN_ATTEMPTS (setting this value to true will enable this feature at the infrastructure level)
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
+  AWS_S3_OIDC_BUCKET
+  AWS_S3_REGION
+  AWS_S3_VERSION
+```
+After these variables have been properly defined, basic information about SAML login attempts will be logged to your S3 bucket. A new CSV file will be created each month so that the logs remain readable. Log storage will take place in a folder structure that looks like this `S3 Bucket > saml_logs > {ENVIRONMENT} > {Network URL hashed though wp_hash function} > {YYYY-MM} > saml_logs.log`.
+
+### AWS CloudWatch Logs
+Define the following envirnoment variables:
+
+ ```
+  LOG_LOGIN_ATTEMPTS (setting this value to true will enable this feature at the infrastructure level)
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
+  AWS_S3_REGION
+  AWS_S3_VERSION
```

Comment:
Ho Man, here's the env variables needed for this features (the same of S3, except for the `AWS_S3_OIDC_BUCKET` one, it is not necessary for CloudWatch integration